src/components/templates/news-and-events/news-and-events.scss

```
@@ -0,0 +1,39 @@
+@use "../../variables" as *;
+
+.templates-news-and-events {
+  background-color: $white_smoke;
+  padding: 56px 0;
+  .organisms-news {
+    padding-left: 0;
+    padding-right: 20px;
+    @media (max-width: $breakpoint-lg) {
+      padding: 0;
+      padding-bottom: 38px;
```

ここわかりづらいのですが35pxになります
```suggestion
      padding-bottom: 35px;
```